app/Models/Bible/BibleFileset.php

```diff
@@ -464,4 +464,25 @@ public static function getDefaultMetaTags() : array
             BibleFilesetTag::STOCK_NO_TAG => null
         ];
     }
+
+    /**
+     * Filter bible fileset records to avoid pulling the old six character text_plain fileset.
```

Comment:
add more to this comment. "... the old six character text_plain fileset when a ten character fileset id exists"

Reply:
@bradflood , It has been updated as you can see in the following line:

https://github.com/faithcomesbyhearing/dbp/pull/865/files#diff-c4f12b64765ea311cd0953e83e7e57a88eb1d01ff5ceb4d584a163f5c949e294R472